lib/chai/core/assertions.js

```diff
@@ -2448,135 +2448,138 @@ module.exports = function (chai, _) {
   Assertion.addMethod('key', assertKeys);
 
   /**
-   * ### .throw([errorLike], [errMsgMatcher], [msg])
+   * ### .throw
+   * ### .throw([errLike[, errMsgMatcher[, msg]]])
+   * ### .throw([errMsgMatcher[, msg]])
    *
-   * When no arguments are provided, `.throw` invokes the target function and
-   * asserts that an error is thrown.
+   * When used as a property assertion or when invoked without any arguments,
+   * `.throw` invokes the target function and asserts that it throws.
    *
-   *     var badFn = function () { throw new TypeError('Illegal salmon!'); };
+   *     var badFn = function () { throw TypeError('Illegal salmon!'); };
    *
+   *     expect(badFn).to.throw;
    *     expect(badFn).to.throw();
    *
-   * When one argument is provided, and it's an error constructor, `.throw`
-   * invokes the target function and asserts that an error is thrown that's an
-   * instance of that error constructor.
+   * When invoked with one argument, and it's a function (presumably a
+   * constructor), `.throw` invokes the target function and asserts that it
```

Comment:
Nit: `s/presumably/typically/`. `presumably` suggests we don't know what it will be, whereas `typically` suggests that is the common pattern.